cli/cmd/sidecar/sidecar.go

```diff
@@ -207,16 +207,14 @@ func initializeDiskCache(env *real_environment.RealEnv) {
 func main() {
 	// Can remove all this configurator stuff once all flags the sidecar uses are
 	// defined outside the configurator.
-	config.RegisterAndParseFlags()
-	configurator, err := config.NewConfiguratorFromData([]byte{})
-	if err != nil {
+	flag.Parse()
+	if err := config.PopulateFlagsFromData([]byte{}); err != nil {
```

Comment:
Do we still need this? Does calling PopulateFlagsFromData with an empty byte slice have any side-effects?

Reply:
nah, good point. this is totally unnecessary.